src/tests/unit/tests/reports/get-quick-assess-summary-model.test.ts

```diff
@@ -75,12 +75,7 @@ describe('getQuickAssessSummaryModel', () => {
             title: 'Automated Checks',
             key: AutomatedChecks.key,
             visualizationType: 0,
-            requirements: [
-                { name: 'Any', key: 'any' },
-                { name: 'Name', key: 'name' },
-                { name: 'Will', key: 'will' },
-                { name: 'Do', key: 'do' },
-            ],
+            requirements: [{ name: 'Any' }, { name: 'Name' }, { name: 'Will' }, { name: 'Do' }],
```

Comment:
This one made me dig a little deeper. The `key` property exists on the outer object (the one defined on line 74), so this appears to be correct

Reply:
I think it might be better to not lose any information and instead adjust line 20 where the type is defined to include `| 'key'`.

Reply:
Added 'key' as per the suggestion.